ipaserver/install/ipa_server_certinstall.py

```diff
@@ -157,6 +157,41 @@ def install_http_cert(self):
         os.chown(os.path.join(dirname, 'key3.db'), 0, pent.pw_gid)
         os.chown(os.path.join(dirname, 'secmod.db'), 0, pent.pw_gid)
 
+    def check_chain(self, pkcs12_filename, pkcs12_pin, nssdb):
+        # create a temp nssdb
+        with NSSDatabase() as tempnssdb:
+            db_password = ipautil.ipa_generate_password()
+            db_pwdfile = ipautil.write_tmp_file(db_password)
+            tempnssdb.create_db(db_pwdfile.name)
+
+            # import the PKCS12 file, then delete all CA certificates
+            # this leaves only the server certs in the temp db
+            try:
+                tempnssdb.import_pkcs12(pkcs12_filename, db_pwdfile.name,
+                    pkcs12_pin)
+                for nickname, flags in tempnssdb.list_certs():
+                    if 'u' not in flags:
+                        tempnssdb.delete_cert(nickname)
+
+            except RuntimeError as e:
+                raise admintool.ScriptError(str(e))
```

Comment:
You don't have to re-raise the exception here, as `RuntimeError` has the same effect as `ScriptError`.